graphene/types/json.py

```diff
@@ -22,7 +23,11 @@ def serialize(dt):
     @staticmethod
     def parse_literal(node):
         if isinstance(node, StringValueNode):
-            return json.loads(node.value)
+            try:
+                return json.loads(node.value)
+            except Exception:
+                raise ValueError("badly formed JSONString")
```

Comment:
Wouldn't it make sense to add the underlying exception message here? Error handling will probably only add"badly formed JSONString" to the query response - I always find it helpful if the error message assists in finding my JSON typo / malformed part when writing queries.